manageiq-providers-openstack.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_runtime_dependency "activesupport",        ">= 5.0", "< 5.3"
+  s.add_runtime_dependency "activesupport",        "~>5.2.4", ">= 5.2.4.3", "< 5.3"
```

Comment:
You can remove the "< 5.3" because that can't happen anyway with "~>5.2.4"

Reply:
Updated.